src/v/config/configuration.cc

```diff
@@ -1115,6 +1115,27 @@ configuration::configuration()
       "Timeout to check if cache eviction should be triggered",
       {.visibility = visibility::tunable},
       30s)
+  , cloud_storage_prefetch_enable(
+      *this,
+      "cloud_storage_prefetch_enable",
+      "Enable prefetch in shadow indexing",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      true)
+  , cloud_storage_prefetch_size_trigger(
+      *this,
+      "cloud_storage_prefetch_size_trigger",
+      "The prefetching of the next segment should be triggered when this "
+      "number or fewer bytes remain to be consumed by the reader from the "
+      "segment",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      32_MiB)
```

Comment:
What's the rationale for the 32MB default?

This would ideally be a function of consumer speed and segment size (that's probably too complex for right now), but the default should reflect some realistic example ratio (e.g. a consider a consumer that runs at 16MB/s for 128MB/s segment sizes -> how early does it have to fetch to hide latency)